src/services/user/registration.js

```diff
@@ -70,9 +70,19 @@ const insertUserToDB = async (app, data, user) => {
 	}
 	return app.service('users').create(user, { _additional: { parentEmail: data.parent_email, asTask: 'student' } })
 		.catch((err) => {
-			logger.warning(err);
+			logger.error(err);
 			// fixme check error message is correct, check err
-			throw new errors.BadRequest('Fehler beim Erstellen des Nutzers. Eventuell ist die E-Mail-Adresse bereits im System registriert.');
+			let msg = 'Fehler beim Erstellen des Nutzers. '
+				+ 'Eventuell ist die E-Mail-Adresse bereits im System registriert. '
+				+ 'Wende dich an den Support. Damit wir dir schnell helfen können, teile uns bitte alle angegebenen E-Mail-Adressen mit.';
+			if (err && err.message) {
+				if (err.message.includes('bereits')) {
+					// account or user exists
+					msg = `${err.message} `
+					+ 'Wahrscheinlich kannst du dich damit bereits einloggen. Nutze dazu den Login. Dort kannst du dir auch ein neues Passwort zusenden lassen.';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 161. Maximum allowed is 120.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203912)